testing/sign_function_test.py

```diff
@@ -0,0 +1,48 @@
+#
+# sign_function_test.py
+# Francois-Michel L Heureux, 2016-06-21
+# This file is part of MLDB. Copyright 2016 Datacratic. All rights reserved.
+#
+
+mldb = mldb_wrapper.wrap(mldb)  # noqa
+
+class SignFunctionTest(MldbUnitTest):  # noqa
+
+    def test_negative_int(self):
```

Comment:
man j'aurais fait un test avec
for input, output in [
    ("-123", -1),
    ("123", 1),
...]:
  self.assertEqual(
     mldb.query("SELECT sign(%s)" % input)[1][1],
     output)